js/stdlib/ts/ALGO.ts

```diff
@@ -1309,9 +1310,11 @@ export const connectAccount = async (networkAccount: NetworkAccount): Promise<Ac
         debug('getState');
         return await getState_(getC, (vibna:BigNumber) => {
           if ( vibne.eq(vibna) ) { return vtys; }
-          const expectedLoc = bin._stateSourceMap[vibne.toNumber().valueOf()];
-          const actualLoc = bin._stateSourceMap[vibne.toNumber().valueOf()];
-          throw Error(`Expected state ${vibne} at ${expectedLoc}, got ${vibna} at ${actualLoc}`);
+          const formatLoc = (l:BigNumber) =>
+            formatAssertInfo(bin._stateSourceMap[l.toNumber()]);
+          const e = formatLoc(vibne);
+          const a = formatLoc(vibna);
+          throw Error(`Expected the Dapp to be at commit() at ${e}, Dapp was actually at commit() at ${a}`);
```

Comment:
DApp

Reply:
I think the print out with the stack in between is really ugly and you should try to make it so the stacks come after (using the pieces of formatAssertInfo)